README.md

```diff
@@ -1 +1,1146 @@
+
+``` 
++-----------+------------+------+-----+---------+-------+
+| Field     | Type       | Null | Key | Default | Extra |
++-----------+------------+------+-----+---------+-------+
+| ID        | int(11)    | NO   | PRI | NULL    |       |
+| Com_Maths | varchar(1) | YES  |     | NULL    |       |
+| Physics   | varchar(1) | YES  |     | NULL    |       |
+| Chemistry | varchar(1) | YES  |     | NULL    |       |
++-----------+------------+------+-----+---------+-------+
+```
+In the below service. it added the student's data through the request. But student's results data must be located in the database as below.
+```
++-----+-----------+---------+-----------+
+| ID  | Com_Maths | Physics | Chemistry |
++-----+-----------+---------+-----------+
+| 100 | A         | A       | A         |
+| 101 | A         | A       | B         |
+| 102 | A         | A       | C         |
+| 103 | A         | B       | A         |
+| 104 | A         | B       | B         |
+| 105 | A         | B       | C         |
+| 106 | A         | C       | A         |
+| 107 | A         | C       | B         |
+| 108 | A         | C       | C         |
++-----+-----------+---------+-----------+
+```
+
+### Developing the service
+
+To implement the scenario, let's start by implementing the message_transformation.bal file, which is the main file in the implementation. This file includes 4 main services as contentfilter, validator, enricher, and backend. Refer to the code attached below. Inline comments are added for better understanding.
+
+> In the below service you have to change below configurations according to your mysql configuration details, 
+>```
+>   host: "localhost",
+>   port: 3306,
+>   username: "root",
+>   password: "wso2123",
+>```
+
+##### passthrough.bal
+
+```ballerina
+
+import ballerina/http;
+import ballerina/log;
+import ballerina/mime;
+import ballerina/io;
+import ballerina/mysql;
+
+endpoint http:Listener contentfilterEP {
+    port:9090
+};
+endpoint http:Listener claimvaditadeEP {
+    port:9094
+};
+endpoint http:Listener contentenricherEP {
+    port:9092
+};
+endpoint http:Listener backendEP {
+    port:9093
+};
+//define endpoints for services
+endpoint http:Client validatorEP {
+    url: "http://localhost:9094/validator"
+};
+endpoint http:Client enricherEP {
+    url: "http://localhost:9092/enricher"
+};
+endpoint http:Client clientEP {
+    url: "http://localhost:9093/backend"
+};
+
+//connect the student details table
+endpoint mysql:Client testDB {
+    host: "localhost",
+    port: 3306,
+    name: "testdb",
+    username: "root",
+    password: "wso2123",
+    poolOptions: { maximumPoolSize: 5 },
+    dbOptions: { useSSL: false }
+};
+
+//connect the student's results details table
+endpoint mysql:Client testDB1 {
+    host: "localhost",
+    port: 3306,
+    name: "testdb1",
+    username: "root",
+    password: "wso2123",
+    poolOptions: { maximumPoolSize: 5 },
+    dbOptions: { useSSL: false }
+};
+
```

Comment:
May be we can have better names for the db endpoints rather than "testDB" and "testDB1" ?

Reply:
Done